src/databricks/labs/ucx/azure/credentials.py

```diff
@@ -278,6 +278,9 @@ def _create_storage_credentials_for_storage_accounts(self) -> list[StorageCreden
 
         execution_results = []
         for access_connector, url in access_connectors:
+            if len(list(self._ws.storage_credentials.list())) > 200:
```

Comment:
hmm, move this into the `create_access_connectors_for_storage_accounts` before the `        if len(used_storage_accounts) == 0:` check

Reply:
Does it make sense to move the check back under migrate_credentials as it will lead to code duplication for Azure and AWS?

Reply:
Addressed the comment and also added limit check for AWS